front/docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
     build:
       dockerfile: ./Dockerfile
       context: ./
+      network: host
```

Comment:
this is probably ok now, but I'll change it to a custom docker network in the next PR (otherwise I don't know how to make frontend API be published since https://docs.docker.com/network/host/  `Note: Given that the container does not have its own IP-address when using host mode networking, port-mapping does not take effect, and the -p, --publish, -P, and --publish-all option are ignored, producing a warning instead:`